.travis.yml

```diff
@@ -10,10 +10,12 @@ compiler:
 env:
   global:
     - INSTALL_DIR=~/mylibs
+    - CMAKE_FLAGS="-DCMAKE_CXX_FLAGS=\"-Werror\" -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR"
```

Comment:
why not simply export CXXFLAGDS="-Werror" at this point and keep install prefix below?